content/2017.07.slack-deb-automatic-updates.rst

```diff
@@ -0,0 +1,123 @@
+Enabling Slack automatic updates on Debian and Ubuntu
+#####################################################
+
+:lang: en
+:date: 2017-07-18
+:url: slack-deb-automatic-updates
+:summary: A little tip to enable automatic updates of Slack with Debian and Linux
+
+
+Installing Slack on Linux
+=========================
+
+The proper way to install Slack is to download the installer for your
+Operating System on the `Slack website <https://slack.com/downloads>`_.
+
+However I realized my slack client was not really up-to-date.
+
+When installing Slack, a daily cronjob is installed automatically
+which is supposed to configure the automatic updates.
```

Comment:
The cronjob isn't really supposed to configure the updates. Maybe `install` or `apply` is better. Plus, the updates themselves aren't automatic, they're being used automatically (i.e. attach "automatic" to the verb rather than the noun).